guides/common/modules/proc_configuring-pull-based-transport-for-remote-execution.adoc

```diff
@@ -0,0 +1,17 @@
+[id="configuring-pull-based-transport-for-remote-execution_{context}"]
+= Configuring Pull-based Transport for Remote Execution
+
+Use this procedure to set up `pull-mqtt` mode for pull-based remote execution transport.
+
+.Procedure
+ifdef::foreman[]
+* Ensure your machine certificates are installed.
```

Comment:
The procedure cannot include bullet points. If this is conceptual information, you should use concept. If it is a procedure, there must be steps. This is very confusing.

Reply:
@vferschm Updated the procedures to be procedures